doc/HOWTO/adding-functions.md

```diff
@@ -0,0 +1,36 @@
+Adding new functions
+====================
+
+This document describes the way to add custom modifications to OpenSSL sources.
+
+ If you are adding new public functions to the custom library build, you need to
+ either add a prototype in one of the existing OpenSSL header files;
+ or provide a new header file and edit
+ [Configurations/unix-Makefile.tmpl](Configurations/unix-Makefile.tmpl)
+ to pick up that file.
+
+ After that perform the following steps:
+
+    ./Configure -Werror --strict-warnings [your-options]
```

Comment:
Please drop -Werror as strict warnings already adds that.

Reply:
removed in ea72ad8